test/controllers/ViewAssessmentSpec.scala

```diff
@@ -43,7 +43,8 @@ class ViewAssessmentSpec extends VoaPropertyLinkingSpec with OptionValues {
     mockDvrCaseManagement,
     StubBusinessRatesValuation,
     mockBusinessRatesAuthorisation,
-    stubMessagesControllerComponents()) {
+    stubMessagesControllerComponents(),
+    false) {
```

Comment:
somehow this false fixes something, but it's not clear how. you could use named parameters, especially for boolean parameters

Reply:
I have added name to Flag is isExternalValuationEnabled